iam_apikey.go

```diff
@@ -0,0 +1,64 @@
+package egoscale
+
+// APIKeyType holds the type of the apikey
+type APIKeyType string
+
+const (
+	// APIKeyUnrestricted is unrestricted
+	APIKeyUnrestricted APIKeyType = "restricted"
+	// APIKeyRestricted is restricted
+	APIKeyRestricted APIKeyType = "unrestricted"
```

Comment:
I suggest naming the constants `APIKeyTypeRestricted`/`APIKeyTypeUnrestricted` so one can know at a glance what they refer to.

Reply:
Also, they are obviously swapped.